extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -376,7 +376,7 @@ export async function makeHttpRequest(account: azdata.Account, subscription: azu
 			break;
 	}
 
-	if (response.status >= 300) {
+	if (response.status <= 200 || response.status >= 299) {
```

Comment:
This is saying 200 and 299 are errors which isn't correct. It should be < 200 || > 299